src/client/game/game-actions.js

```diff
@@ -15,10 +20,37 @@ export function leaveGame() {
 	}
 }
 
-export function setLeader(user) {
+export function setLeader(userObj) {
```

Comment:
userObj also contains some data that isn't necessarily relevant to the user, so we named it differently to make it clear that it isn't the same as a cloak 'user'